cal-adapt/climakitae
# flexible coord drop fn dependent on data source for cdd/hdd

Adds a quick function to provide a flexible coordinate drop for the CDD/HDD final merge step that is flexible for the two different data sources, which have different coordinates to drop. 

---
To test:
Run the INTRO consumption nb in the `dd_nb_fix` branch in cae-notebooks (https://github.com/cal-adapt/cae-notebooks/pull/64) and make sure that data is exported in Step 4g
